neal/sampler.py

```diff
@@ -174,7 +174,7 @@ def sample(self, bqm, beta_range=None, num_reads=None, num_sweeps=1000,
                 with all of the samples and energies found so far.
 
         Returns:
-            :obj:`dimod.Response`: A `dimod` :obj:`~dimod.Response` object.
+            :obj:`dimod.SampleSet`
```

Comment:
You might as well update the sphinx ref type:
```suggestion
            :class:`dimod.SampleSet`
```

Reply:
✔️